Emby.Server.Implementations/Images/BaseDynamicImageProvider.cs

```diff
@@ -60,6 +60,11 @@ public async Task<ItemUpdateType> FetchAsync(T item, MetadataRefreshOptions opti
                 return ItemUpdateType.None;
             }
 
+            if (!options.ReplaceAllImages)
+            {
+                return ItemUpdateType.None;
+            }
```

Comment:
This looks like it will only download images when replace images is checked. What about the case where there are no images already downloaded? I would expect missing images to be downloaded without needing to check the "Replace All Images" box.